lib/matplotlib/tests/test_axes.py

```diff
@@ -8431,3 +8431,15 @@ def test_zorder_and_explicit_rasterization():
     ln, = ax.plot(range(5), rasterized=True, zorder=1)
     with io.BytesIO() as b:
         fig.savefig(b, format='pdf')
+
+
+@mpl.style.context('default')
+def test_rc_axes_label_formatting():
+    mpl.rcParams['axes.labelcolor'] = 'red'
```

Comment:
```suggestion
    mpl.rcParams['axes.labelcolor'] = 'red'
```
I think these should be wrapped in an rcContext, otherwise they apply for the rest of the test file, which could be confusing for future tests.

Reply:
I believe it should be fine because the `matplotlib.style.context` decorator already wraps everything in an `rc_context`:
https://github.com/matplotlib/matplotlib/blob/f726d2cf4e61bc3e01db99bc09a7871d0f574990/lib/matplotlib/style/core.py#L220
Admittedly, I have not tested it.